views/lexicon/tables/datatablex.html.twig

```diff
@@ -40,10 +70,15 @@
                 <td>4 GB</td>
                 <td class="centered">2</td>
                 <td>
-                    {{ html.link({ 'label': 'Reboot', 'class': 'table-action' }) }}<!--
-                    -->{{ html.link({ 'label': 'Suspend', 'class': 'table-action' }) }}<!--
-                    -->{{ html.link({ 'label': 'Shutdown', 'class': 'table-action' }) }}<!--
-                    -->{{ html.link({ 'label': 'Console', 'class': 'table-action' }) }}
+                    {{
+                        html.button_dropdown({
+                            'label': html.icon('ellipsis-h', {'label': 'Actions'}),
```

Comment:
Not sure, if alphabetical order is needed here

Reply:
Not really required